sdk/appconfiguration/app-configuration/test/README.md

```diff
@@ -0,0 +1,62 @@
+# Testing
+
+To run our tests, first do a full update of the project by running `rush update`, then rebuild either this package, with `rush build -t @azure/app-configuration` or all packages with `rush build`, then make sure to build the tests with `rush build:test`. After that, you can either run our unit tests, which won't reach to live services, and our integration tests, which will indeed require existing, reachable resources.
+
+## Run all tests
+
+Run `rushx test:node` or `rushx test:browser` from your terminal to run all the tests under the `./test` folder
+
+## Run all tests in a single test suite
+
+Append the `.only` on the `describe` method corresponding to the test suite.
+
+Then run `rushx test:node` or `rushx test:browser` from your terminal.
+
+## Run a single test
+
+Append the `.only` on the `it` method corresponding to the test.
+
+Then run `rushx test:node` or `rushx test:browser` from your terminal.
+
+## Debug tests using Visual Studio Code
+
+- Select the debug configuration `Debug Unit Tests` from the drop down in the debug viewlet.
+- Add breakpoints in the test you want to debug. Tip: It is helpful to append the `.only` on the test method if you want only that method to be run.
+- Start Debugging
```

Comment:
For debugging using VS Code, the main catch is going to be to ensure you are in the right folder and not root of the repo so that the debug configuration can be picked up. Unless we have one at the root as well which works with all packages. 

I would make the assumption that the reader would know how to work with VS Code and the tip with `.only`. I would keep this short

> Each package has a debug configuration `Debug Unit Tests` that you can use to debug your tests.
Ensure that you have opened the right folder in VS Code so that the corresponding debug configuration can get picked up.

Also, I wouldnt add this section until we first make sure all Track 2 packages do have a working debug configuration.

Reply:
I will remove this section then. Thank you!